Auto rest

shashishailaj opened the pull request:
Pulling

xamarsinx committed:
Building remoteApp SDK ver 0.0.25-preview

dihan0604 committed:
update build

dihan0604 committed:
update build version

Commit:
Clean up the dependencies.

Commit:
Removing unused packages.

Commit:
A few more to remove.

Commit:
Whole mess of package management fail

Cleaned up csproj files, fixed nuget.config and used the right package
replacement variable.

Commit:
remove specifics and old xunit

markcowl committed:
Merge pull request #1831 from begoldsm/AutoRest

Add in DataLake.StoreUploader, fix version

stankovski committed:
Merge pull request #1829 from brjohnstmsft/odata-url-fix

Search SDK: Ensuring that all URL paths are in OData format

huangpf committed:
Merge pull request #325 from Azure/AutoRest

Auto rest

anthony0t committed:
Regenerate the SDK and recapture the session records

stankovski committed:
Merge pull request #1833 from tonytang-microsoft-com/AutoRest

Regenerate the SDK and recapture the session records

huangpf committed:
Merge pull request #329 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

dihan0604 committed:
stats api

hyonholee committed:
Use 0229 AutoRest version

dihan0604 committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

dihan0604 committed:
Stats API

dihan0604 committed:
version

dihan0604 committed:
Stats API

Commit:
Cleanup generated code based on latest spec

This removes unused classes/parameters and updates JobErrorDetails to be
an int vs. string.

Commit:
Update .NET sdk with latest specs

Update the .NET sdk to reflect the latest specs and run tests.
Update version of each package.

amarzavery committed:
Merge pull request #1838 from begoldsm/AutoRest

Cleanup Generated ADL Code Based on Latest Specs

huangpf committed:
Merge pull request #334 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

bitsforthought committed:
Search SDK: Adding missing AnalyzerNames

This is a fix for https://github.com/Azure/azure-sdk-for-net/issues/1842

bitsforthought committed:
Search SDK: Bumping package version to 1.1.1

Commit:
Properly use DateTimeOffset and TimeSpan

Commit:
Merge pull request #1843 from brjohnstmsft/analyzer-fix

Search SDK: Add missing analyzer names

huangpf committed:
Merge pull request #339 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

Commit:
Revert time span change.

hyonholee committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

hyonholee committed:
Add ACS APIs and update client library

hyonholee committed:
Fix ContainserServiceListResult

clamchowder committed:
Add BGP over IPsec APIs with new code generator version

lamchester committed:
Re-add BgpSettings.cs and recorded test data

Commit:
Merge pull request #1841 from lamchester/AutoRest

BGP over IPsec APIs

hyonholee committed:
Update generated files and tests for Compute and ACS

lamchester committed:
Increment version numbers

Commit:
Merge pull request #1850 from lamchester/AutoRest

Increment version numbers

huangpf committed:
Merge branch 'AutoRest' of https://github.com/hyonholee/azure-sdk-for-net into a0312

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

huangpf committed:
Update

huangpf committed:
Update API Version

Commit:
Update to latest Swagger Specs

This gives us:
Custom naming for catalog items and OperationResults
Guids instead of strings

Rerun tests
Generate with latest nightly.

huangpf committed:
Update Test Records

hyonholee committed:
Fix generate.cmd to get the correct json file from Github

hyonholee committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

huangpf committed:
Merge branch 'AutoRest' of https://github.com/hyonholee/azure-sdk-for-net into a0312

huangpf committed:
Update Version

amarzavery committed:
Merge pull request #1848 from hyonholee/AutoRest

Add ACS APIs and update client library

Commit:
update to use latest nightly

This was done to ensure that regenerate did not change any of the code
from when it was generated using my private build.

Commit:
revert change to xproj projectGuid.

markcowl committed:
Merge pull request #1845 from begoldsm/AutoRest

Properly use DateTimeOffset and TimeSpan

huangpf committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into a0312

huangpf committed:
Update

amarzavery committed:
Merge pull request #1853 from huangpf/a0312

AzureCP PR: API Version Update

huangpf committed:
Update Tests

huangpf committed:
Merge branch 'a0312' of https://github.com/huangpf/azure-sdk-for-net into autorest

huangpf committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into autorest

huangpf committed:
Address CRs

amarzavery committed:
Merge pull request #1856 from huangpf/autorest

AzureCP PR - Update Compute Tests

deepakswifty committed:
fix recordings for new api version in network

deepakswifty committed:
fix recordings for compute tests

deepakswifty committed:
resolve commits

x10shun committed:
Add more APIs for the client

deepakswifty committed:
resolve conflicts

deepakswifty committed:
Merge branch 'UpdateNetworkVersion' of https://github.com/DeepakRajendranMsft/azure-sdk-for-net into UpdateNetworkVersion

amarzavery committed:
Merge pull request #1861 from DeepakRajendranMsft/UpdateNetworkVersion

Update network version to 2016-03-30

huangpf committed:
Merge pull request #349 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

amarzavery committed:
Merge pull request #1859 from naveedaz/AutoRest

Some API fixes and enhancements

huangpf committed:
Merge pull request #350 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

calexan committed:
Bringing sdk into sync with doc

calexan committed:
Bringing sdk into sync with doc

dihan0604 committed:
Co-exist API

dihan0604 committed:
Update Version

Commit:
Moving to AutoRest from hydra-spec-pr

dihan0604 committed:
update version

hyonholee committed:
Fix forceUpdateTag parameter to accept any string values.

hyonholee committed:
Update Compute version number to 12.0.1-prerelease

dihan0604 committed:
Update version

Commit:
Update NuGet package version and fix spec file URI in generate.cmd

markcowl committed:
Merge pull request #1871 from hyonholee/AutoRest

Fix forceUpdateTag parameter to accept any string values.

dihan0604 committed:
fix conflict

Commit:
Merge pull request #1872 from dihan0604/AutoRest

Co-existence API and stats API

dihan0604 committed:
Revert "Co-existence API and stats API"

Commit:
Merge pull request #1874 from dihan0604/revert-1872-AutoRest

Revert "Co-existence API and stats API"

dihan0604 committed:
Co exist API with new api version

dihan0604 committed:
Update version

Commit:
Merge pull request #1875 from dihan0604/AutoRest

Co exist API with new api version with new rest API

huangpf committed:
Merge pull request #356 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

Commit:
Revert "Update NuGet package version and fix spec file URI in generate.cmd"

This reverts commit ad67bc3bca56d490c651217d0a361838d2526575.

Commit:
Revert "Moving to AutoRest from hydra-spec-pr"

This reverts commit 6daff77f9cef7bff1fa76756303d4c00ac9c5bf6.

Commit:
Moving to AutoRest from hydra-spec-pr

Commit:
Update NuGet package version and fix spec file URI in generate.cmd

calexan committed:
commenting out problem causing client, will be fixed later

calexan committed:
Updating from PR feedback

dihan0604 committed:
Update sdk

dihan0604 committed:
update version

calexan committed:
Fixing version number in project.json

Commit:
Merge pull request #1864 from calexan/AutoRest

Auto rest

huangpf committed:
Merge pull request #358 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

vinatara committed:
Adding application gateway support in VM ScaleSet

Commit:
Remove subscription id and update enums

Make enum names more useable, re-run tests.

vinatara committed:
Fixed the recording for integrating VMSS and appgw test

vinatara committed:
Added validation for VMScaleSet's NetworkProfile

vinatara committed:
Fixing VMSS network profile validation to treat null and empty list as equal

pinwang81 committed:
Scheduler: support P20 and P10 Premium SKU

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

# Conflicts:

#	src/ResourceManagement/Resource/Microsoft.Azure.ResourceManager/project.json

vinatara committed:
Adding support for NIC and VMSS as application gateway backend.

Commit:
Updates based on new swagger

Need to re-run tests once FE fixes are in.

Commit:
Merge pull request #1888 from vinatara/nrpvmss1

Adding support for NIC and VMSS as application gateway backend.

pinwang81 committed:
Scheduler: change the specUrl path to github repo path instead of local path.

huangpf committed:
Merge pull request #364 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

Commit:
Merge pull request #1882 from vinatara/AutoRest

Adding application gateway support in VM ScaleSet

Commit:
Merge pull request #1886 from pinwang81/AutoRest

Scheduler: support P20 and P10 Premium SKU

huangpf committed:
Merge pull request #365 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

Commit:
Merge pull request #1878 from SiddharthChatrolaMs/AutoRest

moving to auto rest from hydra-spec-pr

huangpf committed:
Merge pull request #366 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

vinatara committed:
Incrementing the version

Commit:
Flatten properties for RedisCache

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

amarzavery committed:
Merge pull request #1890 from SiddharthChatrolaMs/AutoRest

 Flatten properties for RedisCache

Commit:
Merge pull request #1889 from vinatara/nrpvmss1

Incrementing the version

huangpf committed:
Merge pull request #367 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

vivsriaus committed:
Move policy into its own client. Add debugSetting to deployments

Commit:
Add Release Note for redis cache

ankurchoubeymsft committed:
Update ARM-Network SDK with IPv6 and IDNS Properties. Updates SDK generated with updated swagger spec and includes IPv6 and IDNS Properties. Also added test cases and session records for the new properties added. Incremented project and assembly version.

Commit:
Documentation updated

SiddharthChatrolaMs committed:
Update ReleaseNotes.md

Commit:
Merged changes from Pranav related to ReleaseNotes

Commit:
Update SDK and tests with latest

This is latest autorest + latest APIs.

Commit:
Merge pull request #1893 from ankurchoubeymsft/AutoRest

Update ARM-Network SDK with IPv6 and IDNS Properties.

vivsriaus committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AddPolicyClient

huangpf committed:
Merge pull request #369 from Azure/AutoRest

HPF PR: autorest <- Azure:AutoRest

huangpf committed:
Update

amarzavery committed:
Merge pull request #1900 from huangpf/autorest

AzureCP PR for 4/15 Release

ms-alguerra committed:
Swagger based SDK

Commit:
fixes to allow arbitrary character delimiters.

huangpf committed:
Update Version

huangpf committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into autorest

huangpf committed:
Update Version (#1901)

huangpf committed:
Update Assembly Version

ms-alguerra committed:
Fixing version

ms-alguerra committed:
Fixing tests build

Commit:
Update tests after using transfer encoding chunked

Commit:
Merge pull request #1903 from huangpf/autorest

Update Version

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

ms-alguerra committed:
Swagger based SDK

ms-alguerra committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

markcowl committed:
Merge pull request #1902 from t-alguer/AutoRest

Swagger based SDK

x10shun committed:
Add Dynamic Compute Mode

hyonholee committed:
Update SwarmPreview to Swarm for ContainerService GA

Commit:
Merge pull request #1911 from hyonholee/AutoRest

Update SwarmPreview to Swarm for ContainerService GA

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
Fixed dnu restore issue

Commit:
Merge pull request #1912 from hovsepm/AutoRest

Fixed dnu restore issue

emgerner-msft committed:
Storage cool and encryption

emgerner-msft committed:
lock compute storage version

Commit:
Merge pull request #1905 from emgerner-msft/AutoRest

storage encryption & cool

vivsriaus committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AddPolicyClient

vivsriaus committed:
Change api version for deployment tests

vivsriaus committed:
Change api-version of live test records to reflect 2016-02-01

vivsriaus committed:
Fix authZ tests

vivsriaus committed:
Fix role definition test

vivsriaus committed:
Change api-version of partner tests

Commit:
Removed test project build in Release configuration.

Commit:
Merge pull request #1916 from hovsepm/AutoRest

Removed test project build in Release configuration.

vivsriaus committed:
Fake commit

vivsriaus committed:
Disable the failing compute tests for the time being

vivsriaus committed:
Undo SKIPs of VMTests

vivsriaus committed:
Revert "Undo SKIPs of VMTests"

This reverts commit 0f41b71f6edc6a3a2fd02aee7bf0610442a35612.

vivsriaus committed:
Revert "Revert "Undo SKIPs of VMTests""

This reverts commit 1e5ff51acedadba49c85a8b38be6e0c204d68979.

vivsriaus committed:
Revert "Fix role definition test"

This reverts commit 421f918b63a7f903d3414eac1821881de7751d9b.

vivsriaus committed:
Revert "Revert "Fix role definition test""

This reverts commit a214a17c392966b27386048afc8825588be32809.

vivsriaus committed:
Revert "Revert "Revert "Undo SKIPs of VMTests"""

This reverts commit 0b661b3899879f65118c71597913e8e3de891888.

vivsriaus committed:
Revert "Revert "Undo SKIPs of VMTests""

This reverts commit 1e5ff51acedadba49c85a8b38be6e0c204d68979.

vivsriaus committed:
Revert "Undo SKIPs of VMTests"

This reverts commit 0f41b71f6edc6a3a2fd02aee7bf0610442a35612.

vivsriaus committed:
Revert "Disable the failing compute tests for the time being"

This reverts commit 6021e2a254aa9dd733beece173f27774ed275d46.

vivsriaus committed:
Revert "Change api-version of partner tests"

This reverts commit 2d7d10264e1dc48348cc5c856c43858d7af5058a.

Commit:
update tests for ADL SDK

This is a fresh run of all the tests to regenerate all the recordings.

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

vivsriaus committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AddPolicyClient

vivsriaus committed:
Remove ARM project from Cdn and add package dependency instead

vivsriaus committed:
Add package dependency for ARM, instead of project

vivsriaus committed:
Revert vs version change from sln files

vivsriaus committed:
Fix compute sln

xamarsinx committed:
Remove RemoteApp SDK

vivsriaus committed:
Fix ARM test mocks

vivsriaus committed:
Remove ARM dependency from Storage

vivsriaus committed:
Update to 2.0.0

Commit:
Merge pull request #1892 from vivsriaus/AddPolicyClient

Move policy into its own client. Add debugSetting to deployments. Remove project dependency on ARM, add package dependency instead

vivsriaus committed:
Update ARM assembly and file version to 2.0.0

Commit:
Merge pull request #1918 from begoldsm/AutoRest

ADL Updates

vivsriaus committed:
Update version in project.json

vivsriaus committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AddPolicyClient

Commit:
Merge pull request #1895 from SiddharthChatrolaMs/AutoRest

Release notes added for Redis Management Client

vivsriaus committed:
Merge pull request #1925 from vivsriaus/AddPolicyClient

Update ARM assembly and file version to 2.0.0

Commit:
Merge pull request #1908 from naveedaz/AutoRest

Add Dynamic Compute Mode

begoldsm committed:
Update ADLS uploader. (#1926)

merging due to low risk.

bitsforthought committed:
Search SDK: Adding test coverage for merge of null fields in static model

This is in response to the following issue:

https://github.com/Azure/azure-sdk-for-net/issues/1804

The new test shows how to set NullValueHandling on a per-property basis to
allow for merging of null values.

vivsriaus committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AddPolicyClient

vivsriaus committed:
Update ARM assembly to 3.0.0 so the powershell solution can still use the old hyak client (2.0.0) and this new one (3.0.0)

Commit:
Merge pull request #1929 from vivsriaus/AddPolicyClient

Update ARM assembly to 3.0.0

bitsforthought committed:
Search SDK: Fixing errant comment

Commit:
Merge pull request #1928 from brjohnstmsft/nullhandling-fix

Search SDK: Adding test coverage for merge of null fields in static m…

markcowl committed:
Merge pull request #1922 from amarsi/AutoRest

Remove RemoteApp SDK

Commit:
New operation for reboot tenant

Commit:
Updating minor version as new operation is added

Commit:
Adding shard Id parameter and add description that data loss is possible

Commit:
Added comment that it can cause data loss

bitsforthought committed:
Search SDK: Updating to target version 2015-02-28-Preview of the REST API

Changes:

- Updating ApiVersion to 2015-02-28-Preview
- Updating NuGet package version to 2.0.0-preview
- Updating assembly version to 2.0.0.0
- Pointing codegen scripts at new Swagger specs
- Added new StatusCode property to IndexingResult
- Updated all session records; All tests pass with the new API version

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

markcowl committed:
Rename ARM ResourceManager Management Package

Commit:
Added doc generation to Microsoft.Azure.Management.Storage

Commit:
Merge pull request #1937 from hovsepm/AutoRest

Added doc generation to Microsoft.Azure.Management.Storage

vivsriaus committed:
Rename ARM package, fix sln, change assembly to 1.0.0

vivsriaus committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into armrename

bitsforthought committed:
Search SDK: Fixing IndexBatchException.FindFailedActionsToRetry

Taking advantage of the new StatusCode property on IndexingResult to correctly
classify errors as retryable or not.

markcowl committed:
Merge pull request #1938 from vivsriaus/armrename

Rename ARM autorest assembly to Microsoft.Azure.Management.ResourceManager

vivsriaus committed:
Fix vs version in sln

vivsriaus committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into armrename

vivsriaus committed:
Merge pull request #1940 from vivsriaus/armrename

Fix vs version in sln for ARM package

vivsriaus committed:
Rename ARM namespace to Microsoft.Azure.Management.ResourceManager

vivsriaus committed:
Use new NS in test files

vivsriaus committed:
Change version to 1.1.0

Commit:
Merge pull request #1933 from brjohnstmsft/preview-upgrade

Search SDK: Updating to target version 2015-02-28-Preview of the REST…

Commit:
Merge pull request #1942 from vivsriaus/armrename

Rename ARM namespace to Microsoft.Azure.Management.ResourceManager

Commit:
Merge pull request #1943 from brjohnstmsft/retry-fix

Search SDK: Fixing IndexBatchException.FindFailedActionsToRetry

markcowl committed:
Merge pull request #1935 from SiddharthChatrolaMs/AutoRest

 Adding new operation for rebooting tenant

vivsriaus committed:
Remove RMError, add missing properties to RMErrorWithDetails

vivsriaus committed:
Update RM version

markcowl committed:
Merge pull request #1946 from vivsriaus/FixError

Remove RMError, add missing properties to RMErrorWithDetails

dihan0604 committed:
address merge issue

dihan0604 committed:
[Bug fix] Remove extra slash to avoid NRP parsing issue and to keep api signature consistent

dihan0604 committed:
Update version

bitsforthought committed:
Search SDK: Moving from csproj and packages.config to xproj and project.json

Changes:

  - Updated code to be able to compile for .NET 4.5, DNX, and PCL profile 111
  - Replaced project files and updated .sln file
  - Updated dependency on Microsoft.Spatial to 6.15, which now supports
    profile 111
  - Changed TestFramework and friends from NuGet package dependency to
    project dependency and updated usage accordingly

bitsforthought committed:
Search SDK: Updating all session records after DNX toolchain changes

bitsforthought committed:
Search SDK: Removing leftover .csproj and fixing build task

Commit:
Merge pull request #1948 from brjohnstmsft/dnx-pr

Search SDK: Moving from csproj and packages.config to xproj and project.json

dihan0604 committed:
StatsAPI

dihan0604 committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

dihan0604 committed:
update version

Commit:
Merge pull request #1880 from dihan0604/AutoRest

SDK change due to new swagger change for stats API

azuresdkci commented:
Can one of the admins verify this patch?

azurecla commented:
Hi __@shashishailaj__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

smithaborkar committed:
notificationHub changes using swagger file

smithaborkar committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
Fixed dnx build issue for empty test list.

begoldsm committed:
Fix missing functionality (#1953)

* One missed change.

* Update the version of DataLake.Store

Prepping the latest version to require the latest version of the ADLS
package and refactoring the code as necessary.

* Add table partition support: NEXT RELEASE.

This also updates documentation.

* Update to include delete all secrets.

* Remove partitions temporarily.

Need to remove these for a quick update to the shipped functionality.

* missed an assert.

Commit:
Merge pull request #1954 from hovsepm/AutoRest

Fixed dnx build issue for empty test list.

Commit:
Fixed dnx test issue for empty test list.

Commit:
Merge pull request #1956 from hovsepm/AutoRest

Fixed dnx test issue for empty test list.

hovsepm commented:
@shashishailaj you should not pull from AutoRest branch and then create a commit to master branch. Please reopen this PR to Autorest - if it is autorest based client, or master if it is hydra/hyak based client.
